config/config.go

```diff
@@ -49,6 +49,12 @@ type UpdateStruct struct {
 	// - "self-signed", creating TLS certificates if needed
 	UseTLS string `json:",omitempty"` // -tls
 
+	// NoPassword, if true, prevents any password from being included in the
+	// image. The only access will be over the serial console and the web
+	// interface will only accept connections from localhost, which userspace
+	// programs can choose to tunnel in.
```

Comment:
nit: I’m not familiar with the phrasing of “tunneling in localhost”. Would you mind rephrasing this? Maybe something like “which userspace programs can make available with their own authentication check (e.g. Tailscale)”?

Reply:
Done. PTAL.